Resources/Prototypes/Entities/Objects/Fun/bike_horn.yml

```diff
@@ -39,3 +39,20 @@
     damage:
       types:
         Blunt: 0
+
+- type: entity
+  parent: BikeHorn
+  id: GoldenBikeHorn
```

Comment:
make this worth more than the normal bike horn

Reply:
i changed the value to 1000, not crazy amount considering rarity.